### PR TITLE
opencensus: add metrics quickstart

### PR DESCRIPTION
**Pull request by tbpg**

### CODE REVIEW

File: opencensus/metrics_quickstart/main.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+func main() {
+	if projectID == "" {
+		log.Fatal("The GOOGLE_CLOUD_PROJECT environment variable must be set.")
+	}
+
+	ctx := context.Background()
+
+	// Register the view. It is imperative that this step exists,
+	// otherwise recorded metrics will be dropped and never exported.
+	if err := view.Register(latencyView); err != nil {
```
Comment (rakyll) [inline opencensus/metrics_quickstart/main.go]: Create this view inline. You don't need this view elsewhere.

if err := view.Register(&view.View{
		Name:        "demo/latency",
		Measure:     latencyMs,
		Description: "The distribution of the latencies",
 		// Latency in buckets:
		// [>=0ms, >=100ms, >=200ms, >=400ms, >=1s, >=2s, >=4s]
		Aggregation: view.Distribution(0, 100, 200, 400, 1000, 2000, 4000),
	}); err != nil {

File: opencensus/metrics_quickstart/main.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		// [>=0ms, >=100ms, >=200ms, >=400ms, >=1s, >=2s, >=4s]
+		Aggregation: view.Distribution(0, 100, 200, 400, 1000, 2000, 4000),
+	}
+)
+
+var projectID = os.Getenv("GOOGLE_CLOUD_PROJECT")
+
+func main() {
+	if projectID == "" {
+		log.Fatal("The GOOGLE_CLOUD_PROJECT environment variable must be set.")
```
Comment (rakyll) [inline opencensus/metrics_quickstart/main.go]: stackdriver.NewExporter resolves the project ID from the ADC, this env var is not required.

File: opencensus/metrics_quickstart/main.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	}
+
+	// Enable OpenCensus exporters to export metrics
+	// to Stackdriver Monitoring.
+	// Exporters use Application Default Credentials to authenticate.
+	// See https://developers.google.com/identity/protocols/application-default-credentials
+	// for more details.
+	exporter, err := stackdriver.NewExporter(stackdriver.Options{
+		ProjectID:    projectID,
+		MetricPrefix: "my-metrics",
```
Comment (rakyll) [inline opencensus/metrics_quickstart/main.go]: Please avoid this prefix, metric prefix is not something most users will need.

File: opencensus/metrics_quickstart/main.go [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+	"go.opencensus.io/stats/view"
+	"golang.org/x/exp/rand"
+)
+
+var (
+	// The latency in milliseconds.
+	latencyMs = stats.Float64("demo/latency", "The latency in milliseconds", "ms")
+
+	latencyView = &view.View{
+		Name:        "demo/latency",
```
Comment (rakyll) [inline opencensus/metrics_quickstart/main.go]: The name of the metric should represent what latency this is? Request latency, task latency, etc?

Comment (tbpg) [reply opencensus/metrics_quickstart/main.go]: Done.

**Pull request merged**